### PR TITLE
[Containerapp] `az containerapp env create`: Support `--dapr-connection-string` and release version `1.0.0b1`

### PR DESCRIPTION
**Pull request by Greedygre**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

We supported `az containerapp env create --dapr-connection-string` in azure-cli at 2024-05 with PR https://github.com/Azure/azure-cli/pull/28625/files. 
Sync this argument to containerapp extension .

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [ ] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Containerapp

**Review by Juliehzl — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Comment by yanzhudd** [comment]
please fix the CI issues.

**Comment by Greedygre** [comment]
Hi @yonzhan 

Seems something breaking connecedk8s due to:
>   from azure.cli.core.auth.msal_authentication import ServicePrincipalCredential
E   ModuleNotFoundError: No module named 'azure.cli.core.auth.msal_authentication'

src/connectedk8s/azext_connectedk8s/_clientproxyutils.py:21: ModuleNotFoundError

Can you help to check? Thanks!
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

    import sys
    import os
    import platform
    import base64
    import json
    import requests
    import yaml
    import time
    import azext_connectedk8s._constants as consts
    from base64 import b64encode, b64decode
    from azure.cli.core._profile import Profile
    from azure.cli.core import telemetry
    from azure.cli.core.azclierror import CLIInternalError
    from psutil import process_iter, NoSuchProcess, AccessDenied, ZombieProcess, net_connections
    from knack.log import get_logger
>   from azure.cli.core.auth.msal_authentication import ServicePrincipalCredential
E   ModuleNotFoundError: No module named 'azure.cli.core.auth.msal_authentication'

src/connectedk8s/azext_connectedk8s/_clientproxyutils.py:21: ModuleNotFoundError
- generated xml file: /home/cloudtest/.azdev/env_config/mnt/vss/_work/1/s/env/test_results.xml -
=========================== short test summary info ============================
FAILED src/containerapp/azext_containerapp/tests/latest/test_containerapp_up_on_connected_env.py::ContainerAppUpConnectedEnvImageTest::test_containerapp_up_on_arc_e2e
```

**Review by LaylaLiu-gmail — APPROVED** [review]

File: src/containerapp/HISTORY.rst [hunk 1]
```diff
@@ -4,9 +4,12 @@ Release History
...
 upcoming
 ++++++
-* 'az containerapp sessionpool create/update': Add NodeLTS container-type.
 
-* 'az containerapp env java-component`: Support --min-replicas and --max-replicas for Java components'
+1.0.0b1
+++++++
+* 'az containerapp sessionpool create/update': Add NodeLTS container-type.
+* 'az containerapp env java-component': Support --min-replicas and --max-replicas for Java components
+* 'az containerapp env create': Support `--dapr-connection-string` to set application insight connection string
```
Comment (Juliehzl) [inline src/containerapp/HISTORY.rst:12]: Is it supported in main? why do you need to add it in extension?

Comment (Greedygre) [reply src/containerapp/HISTORY.rst:12]: We need to add argument in extension's `create_managed_environment`. Manually sync argument.

**Review by Juliehzl — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**